### ISSUE TITLE
Data sources' "show data from" option works inconsistently

### ISSUE THREAD

**Issue by joerievers** [CNugteren/NLWeer#16]
It appears to me the data sources' "show data from (KNMI/Buienradar) option only hides the views (from the corresponding data source) you have selected from the sidebar and First Screen selector.

There are a couple of problems with this implementation:

- You can set the first screen shown to a map from a data source and view you have enabled and after that disable the data source. The map will still load the map from a source you have disabled when you open the app, this will lead to a confusing UX when you open the First Screen option, as the map it loads first will be none of the available items in the First Screen selection menu. 

- The navigation buttons cycle through all selected views, even from ones for which you have the data source disabled. When the user wants to disable the view confusingly enough the user has to first enable the data source, then disable the view, and if the users prefers not to use a data source, disable the data source again.

**Comment by CNugteren** [comment]
Thanks for your feedback. I'm not really sure about whether your first point is an actual issue, but the second one definitely is a bug that I will address soon: the new navigation buttons shouldnt cycle through disabled data-sources.

**Comment by CNugteren** [comment]
I have now addressed both issues:
```
commit 41678f58c3614a9ebc78661f28c6dba3bc946458
Author: Cedric Nugteren <web@cedricnugteren.nl>
Date:   Thu Aug 13 21:30:07 2020 +0200

    Resets first screen to empty for disabled maps
    
    When a individual map or an entire source is disabled, the
    first screen is now reset to the empty screen automatically.

commit 7bace839b2f729e49d4a3b66a4c09b8b68966541
Author: Cedric Nugteren <web@cedricnugteren.nl>
Date:   Thu Aug 13 21:00:36 2020 +0200

    New navigation button no longer navigates to disabled sources
    
    The new optional navigation buttons used to only check if a certain
    map is enabled/disabled, but it would still cycle through entire
    disabled data-sources. This is now fixed: it now checks for both.
```
This will be part of a soon to be released 1.4.1 release.

**Comment by CNugteren** [comment]
Closing this, as this fix is now released as part of 1.4.1. Will be available on F-Droid automatically in a few days.
